src/shared/otterizecloud/otterizecloudclient/cloud_client.go

```diff
@@ -31,9 +33,13 @@ func NewClient(ctx context.Context) (graphql.Client, bool, error) {
 		AuthStyle:    oauth2.AuthStyleInParams,
 	}
 
-	tokenSrc := cfg.TokenSource(ctx)
-	graphqlUrl := fmt.Sprintf("%s/graphql/v1beta", apiAddress)
-	httpClient := oauth2.NewClient(ctx, tokenSrc)
+	// Timeout for oauth token acquisition is set using the http client passed to the token source context
```

Comment:
Right now it might sound like this timeout is only relevant to oauth, make it clearer that this timeout is for all HTTP calls (including oauth)